tpl/bootstrap.php

```diff
@@ -157,9 +181,9 @@
 foreach ($EXPIRE as $key => $value):
 ?>
 							<option value="<?php echo $key; ?>"<?php
-    if ($key == $EXPIREDEFAULT):
+	if ($key == $EXPIREDEFAULT):
```

Comment:
I guess these changes happened accidentally. For PHP code we use 4 spaces indention, HTML uses tabs. If you want to use tabs only in the templates, please update page.php accordingly, so that we stay consistent.

Reply:
Ah well… but let's not mix these in one file.
Basically we said to said to treat templates like HTML files and thus use [tabs](https://github.com/PrivateBin/PrivateBin/blob/master/tpl/.editorconfig#L7). Of course that line is PHP code, but well… not really a good idea to mix tabs and spaces.